code/modules/client/preference_setup/loadout/loadout_fluffitems.dm

```diff
@@ -0,0 +1,840 @@
+	character_name = list("Cassandra Selone")
+
+/datum/gear/fluff/kilano_dress
+	path = /obj/item/clothing/under/dress/fluff/kilano
+	ckeywhitelist = list("beyondmylife")
+	character_name = list("Kilano Soryu")
+
+/datum/gear/fluff/kilano_gloves
+	path = /obj/item/clothing/gloves/fluff/kilano
+	ckeywhitelist = list("beyondmylife")
+	character_name = list("Kilano Soryu")
+
+/datum/gear/fluff/netra_box
+	path = /obj/item/weapon/storage/box/fluff/kilano
+	ckeywhitelist = list("beyondmylife")
+	character_name = list("Ne'tra Ky'ram")
+	allowed_roles = list("Colony Director", "Head of Personnel", "Security Officer", "Warden", "Head of Security","Detective")
+
+/datum/gear/fluff/xin_sovietuniform
+	path = /obj/item/clothing/under/soviet
+	ckeywhitelist = list("britishrabbit")
+	character_name = list("Xin Xiao")
+	description = "This soviet uniform has seen considerable use over the years, it's rather worn in some places, frayed in others and the stomach region has signs of being stretched out repeatedly."
+
+/datum/gear/fluff/tasald_box
+	path = /obj/item/weapon/storage/box/fluff/tasald
+	ckeywhitelist = list("bwoincognito")
+	character_name = list("Tasald Corlethian")
+
+/datum/gear/fluff/octavius_box
+	path = /obj/item/weapon/storage/box/fluff/octavious
+	ckeywhitelist = list("bwoincognito")
+	character_name = list("Octavious Ward")
+
+/datum/gear/fluff/jayda_meduniform
+	path = /obj/item/clothing/under/solgov/utility/sifguard/medical/fluff
+	ckeywhitelist = list("burritojustice")
+	character_name = list("Jayda Wilson")
+
+//  C CKEYS
+/datum/gear/fluff/james_disk
+	path = /obj/item/weapon/disk/data
+	ckeywhitelist = list("cockatricexl")
+	character_name = list("James Holder")
+
+/datum/gear/fluff/jasmine_implant
+	path = /obj/item/weapon/implanter/reagent_generator/jasmine
+	ckeywhitelist = list("cameron653")
+	character_name = list("Jasmine Lizden")
+
+/datum/gear/fluff/diana_robe
+	path = /obj/item/clothing/suit/fluff/purp_robes
+	ckeywhitelist = list("cameron653")
+	character_name = list("Diana Kuznetsova")
+
+/datum/gear/fluff/diana_tiara
+	path = /obj/item/clothing/head/fluff/pink_tiara
+	ckeywhitelist = list("cameron653")
+	character_name = list("Diana Kuznetsova")
+
+/datum/gear/fluff/aika_coat
+	path = /obj/item/clothing/suit/fluff/blue_trimmed_coat
+	ckeywhitelist = list("chaoko99")
+	character_name = list("Aika Hisakawa")
+
+/datum/gear/fluff/sariU_disk
+	path = /obj/item/weapon/disk/limb/eggnerdltd
+	ckeywhitelist = list("cross_exonar")
+	character_name = list("Sari-U")
+
+/datum/gear/fluff/sariE_disk
+	path = /obj/item/weapon/disk/limb/eggnerdltd
+	ckeywhitelist = list("cross_exonar")
+	character_name = list("Sari-E")
+
+//  D CKEYS
+/datum/gear/fluff/dhaeleena_medal
+	path = /obj/item/clothing/accessory/medal/silver/security/fluff/dhael
+	ckeywhitelist = list("dhaeleena")
+	character_name = list("Dhaeleena M'iar")
+
+/* Item not defined.
+/datum/gear/fluff/elliot_medal
+	path = /obj/item/clothing/accessory/medal/medical
```

Comment:
Someone should probably go ahead and add the missing medals. I will talk to Admins about this.

Reply:
Removed it, as this character hasn't been seen (to my knowledge) for a long time. Certainly long enough where their missing medal wasn't noticed